ignite/handlers/checkpoint.py

```diff
@@ -26,6 +26,9 @@ class ModelCheckpoint(object):
             an `ignite.engines.Engine` object,
             and return a score (`float`). Objects with highest scores will be retained.
             Exactly one of (`save_interval`, `score_function`) arguments must be provided.
+        score_name (str, optional):
+            if `score_function` not None, it is possible to store its absolute value using `score_name`. See Notes for
```

Comment:
why do we want to store the abs value and not the actual value?

Reply:
as `score_function` can return `-loss` and as objects with highest scores are retained, so to avoid sign minus in the filename

Reply:
Ah ok, I see your point now. cool LGTM - thanks @vfdev-5!